docs/sources/developers/http_api/dashboard.md

```diff
@@ -253,6 +253,93 @@ Status Codes:
 - **403** – Access denied
 - **404** – Not found
 
+## Hard delete dashboard by uid
+
+`DELETE /api/dashboards/uid/:uid/trash`
+
+Will delete permanently the dashboard given the specified unique identifier (uid).
+
+**Required permissions**
+
+See note in the [introduction]({{< ref "#dashboard-api" >}}) for an explanation.
+
+| Action              | Scope                         |
+| ------------------- | ----------------------------- |
+| `dashboards:delete` | `dashboards:*`<br>`folders:*` |
+
+**Example Request**:
+
+```http
+DELETE /api/dashboards/uid/trash/cIBgcSjkk HTTP/1.1
```

Comment:
this example contradicts the above, shouldn't it be:

```
DELETE /api/dashboards/uid/cIBgcSjkk/trash HTTP/1.1
```

Reply:
Good catch! Fixed PATCH too.